bigbluebutton-html5/public/locales/en.json

```diff
@@ -696,6 +696,7 @@
     "app.lock-viewers.button.cancel": "Cancel",
     "app.lock-viewers.locked": "Locked",
     "app.lock-viewers.unlocked": "Unlocked",
```

Comment:
Is this used anywhere else?

Reply:
It's been removed in the latest commit. Wasn't being used anywhere else.